LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 
```

Comment:
Should we list any [`<copyright holders>`](http://opensource.org/licenses/MIT) after the year? @sfkeller If so, what should they be?
* **HSR Hochschule für Technik, Rapperswil**<br />or
* **Geometa Lab at HSR Hochschule für Technik, Rapperswil**<br />or
* **Institut für Software der HSR Hochschule für Technik, Rapperswil**<br />or rather just
* **osmaxx contributers**<br />so that we don't have to change this line when accepting contributions from outside HSR?